doctor/lib/text_extraction.py

```diff
@@ -0,0 +1,326 @@
+import re
+
+import pdfplumber
+from pdfplumber.ctm import CTM
+import pytesseract
+from pytesseract import Output
+import pandas as pd
+from PIL import Image
+
+
+def deskew(obj: dict) -> bool:
+    """Remove skewed text from a page
+
+    CTM stands for current transformation matrix.
+    Pdf plumber has a method to calculate the angle of text which we use here
+
+    Traditionally this is only seen in circular stamps which confuses the
+    content, or in perpendicular text of the ninth circuit courts which also
+    confuses the text.
+
+    :param obj: dictionary from pdfplumber for each word
+    :return: if the text should be returned
+    """
+    if (matrix := obj.get("matrix")) is None:
+        return True
+
+    # Remove Skew
```

Comment:
Again, this doesn't deskew anything, right? It checks if it's skewed? Therefore this should be better, right?

```suggestion
def is_skewed(obj: dict) -> bool:
    """Check if a PDF plumber dict is skewed.

    CTM stands for current transformation matrix.

    Pdf plumber has a method to calculate the angle of text which we use here

    Traditionally this is only seen in circular stamps which confuses the
    content, or in perpendicular text of the ninth circuit courts which also
    confuses the text.

    :param obj: dictionary from pdfplumber for each word
    :return: if the text should be returned
    """
    if (matrix := obj.get("matrix")) is None:
        return True

    # Is the content skewed?
```